tools/SetupFlow/DevHome.SetupFlow/Models/CloneRepoTask.cs

```diff
@@ -232,7 +248,28 @@ IAsyncOperation<TaskFinishedState> ISetupTask.Execute()
                 return TaskFinishedState.Failure;
             }
 
+            SummaryScreenInformation = _host.GetService<CloneRepoSummaryInformationViewModel>();
+
+            // Search for a configuration file.
+            var configurationDirectory = Path.Join(_cloneLocation.FullName, _configurationFolderName);
+            if (Directory.Exists(configurationDirectory))
+            {
+                var fileToUse = Directory.EnumerateFiles(configurationDirectory)
+                .Where(file => file.EndsWith(_configurationFileYamlExtension, StringComparison.OrdinalIgnoreCase) ||
+                               file.EndsWith(_configurationFileWingetExtension, StringComparison.OrdinalIgnoreCase))
+                .OrderBy(configurationFile => File.GetLastWriteTime(configurationFile))

```

Comment:
>OrderBy [](http://example.com/codeflow?start=17&length=7)

should this be OrderByDescending to get the newer config file?

Reply:
True.  Thank you.